nix/status-go/mobile/build.nix

```diff
@@ -47,7 +47,7 @@ in buildGoPackage {
       -ldflags="$ldflags" \
       -target=${concatStringsSep "," targets} \
       ${optionalString isAndroid "-androidapi=${platformVersion}" } \
-      ${optionalString isIOS "-iosversion=${platformVersion}" } \
+      ${optionalString isIOS "-iosversion=${platformVersion} -tags=netgo" } \
```

Comment:
What does this flag do?

Reply:
The `netgo` build tag will tell the Go compiler to use `cgo` for networking instead of `Go`.
This resolves the build issue with newly updated `gomobile` and `golang` version.

Reply:
Does it make sense only for iOS?

Reply:
Yeah only the iOS build would fail because of go upgrade and without this flag

Reply:
Okay, but what would happen if we used if to Android too?

Reply:
not sure, but attempted to see how things work -> https://github.com/status-im/status-mobile/pull/18564/commits/a86d815bd29c89d27f76290a2f5305c40a8780d1

Reply:
builds are green so i guess its fine to apply this tag to both android and iOS, seems more consistent that way!
Thank you.

Reply:
Cool.